### DISCUSSION TITLE
Disable rotating by keyboard

### DISCUSSION THREAD

**Discussion by QusaiAlghalil** [visgl/deck.gl#8911] [Q&A]
Hi, in the controller props I set the dragRotate to false and it works for the mouse events, but still able to rotate by keyboard.
If I set the keyboard to false it will disable all the keyboard actions,
Is there any way to disable just the rotating action by keyboard and keep other actions ?

**Comment by Pessimistress** [comment]
You can [implement a custom controller](https://deck.gl/docs/api-reference/core/controller#example-implementing-a-custom-controller) and intercept the `keydown` event.
